packages/apps/reputation-oracle/server/src/modules/storage/storage.service.spec.ts

```diff
@@ -108,6 +105,10 @@ describe('StorageService', () => {
     jest.spyOn(pgpConfigService, 'encrypt', 'get').mockReturnValue(true);
   });
 
+  beforeEach(() => {
+    downloadFileFromUrlSpy = jest.spyOn(StorageService, 'downloadFileFromUrl');
```

Comment:
Mocks in this test file are completely ruined and conflict with each other, so defining new spy before each test is needed to as a temp solution. We are going to revisit reputation oracle, tests will be part of that